snapshot/git/gitdb/db.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/twitter/scoot/snapshot/store"
 )
 
+// TODO The interfaces and functionality here should be refactored to only use git when necessary.
```

Comment:
we might want to revisit the overall snapshot/bundlestore implementation - does remexec/cloud exec need all the snapshots (fake_checkouter, fake_filer, etc.), gitdb, bundlestore pieces?  Can we implement a simpler bundlestore server (it seems the bazel server is a much simpler implementation?)

Reply:
Agree but this is going to be complicated and time consuming to unravel. I would like us to eventually dedicate an amount of time specifically for cleaning up this repo.